corehq/apps/userreports/templates/userreports/partials/report_filters.html

```diff
@@ -0,0 +1,19 @@
+{% load i18n %}
+{% load userreports_tags %}
+
+<div class="hq-report-filters">
+  {% if report_filters %}
+    {% for filter in report_filters %}
+      <div class="mb-3">
+        <h4 class="form-label">{{ filter.label }}</h4>
+        {% with filter_context|get:filter.css_id as context_ %}
```

Comment:
can you share what is happening here?

Reply:
Oh! I didn't realize, we [do this already over here](corehq/apps/reports_core/templates/reports_core/partials/bootstrap5/filter_panel.html:15)! I'll use that template tag when I refactor this.